libs/books/data-access/src/lib/+state/books.effects.ts

```diff
@@ -5,14 +5,15 @@ import { catchError, map, switchMap } from 'rxjs/operators';
 import { Actions, createEffect, ofType } from '@ngrx/effects';
 import { Book } from '@tmo/shared/models';
 import * as BooksActions from './books.actions';
+import { API_ENDPOINT } from '@tmo/books/data-access';
 
 @Injectable()
 export class BooksEffects {
   searchBooks$ = createEffect(() =>
     this.actions$.pipe(
       ofType(BooksActions.searchBooks),
       switchMap((action) =>
-        this.http.get<Book[]>(`/api/books/search?q=${action.term}`).pipe(
+        this.http.get<Book[]>(API_ENDPOINT.GET_BOOKS_BY_QUERY+action.term).pipe(
```

Comment:
While this works, I personally don't find it as readable as the previous version. It's also less flexible. How would you implement this if there were multiple query params?

Reply:
reverted